%YAML:1.0
---
model_type: MEI
camera_name: mei
image_width: 640
image_height: 480
mirror_parameters:
   xi: 0.09557007954305405
distortion_parameters:
   k1: 0.10826611685860128
   k2: -0.09923294297453557
   p1: 0.0011623906355173333
   p2: -0.002079607891631393
projection_parameters:
   gamma1: 316.119648816083
   gamma2: 319.33004694612106
   u0: 321.12477968361753
   v0: 243.35550767194374
